tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedLinagoraSecondaryBlobStoreTest.java

```diff
@@ -256,4 +267,39 @@ void sendEmailShouldResultingInEventuallySavingDataToBothObjectStoragesWhenSecon
                 assertThat(blobIds2).hasSameElementsAs(blobIds);
             });
     }
+
+    @Test
+    void sendEmailShouldResultingInEventDeadLettersWhenSecondStorageIsDownForLongTime(GuiceJamesServer server) throws Exception {
+        secondaryS3.pause();
+
+        given()
+            .body(LinagoraEmailSendMethodContract$.MODULE$.bobSendsAMailToAndre(server))
+        .when()
+            .post()
+        .then()
+            .statusCode(HttpStatus.SC_OK)
+            .contentType(JSON)
+            .extract()
+            .body()
+            .asString();
+
+        Thread.sleep(ONE_MINUTE);
+
+        BlobStoreProbe blobStoreProbe = server.getProbe(BlobStoreProbe.class);
+        BucketName bucketName = Flux.from(blobStoreProbe.getPrimaryBlobStoreDAO().listBuckets()).collectList().block().getFirst();
+        Group group = new FailedBlobOperationListener.FailedBlobOperationListenerGroup();
+
+        calmlyAwait.atMost(TEN_SECONDS)
+            .untilAsserted(() -> {
+                List<BlobId> expectedBlobIds = Flux.from(blobStoreProbe.getPrimaryBlobStoreDAO().listBlobs(bucketName)).collectList().block();
+                List<EventDeadLetters.InsertionId> insertionIds = blobStoreProbe.getEventDeadLetters().failedIds(group)
+                    .collectList().block();
+                List<FailedBlobEvents.BlobAddition> events = insertionIds.stream()
+                    .map(insertionId -> blobStoreProbe.getEventDeadLetters().failedEvent(group, insertionId).block())
+                    .map(FailedBlobEvents.BlobAddition.class::cast)
+                    .toList();
+                assertThat(bucketName).isEqualTo(events.getFirst().bucketName());
+                assertThat(events.stream().map(FailedBlobEvents.BlobAddition::blobId)).hasSameElementsAs(expectedBlobIds);
+            });
+    }
```

Comment:
No what I whish here is that we reprocess them (webadmin call) and ensure they get added in the second blob store.